src/ray/common/ray_config_def.h

```diff
@@ -157,7 +157,7 @@ RAY_CONFIG(size_t, free_objects_batch_size, 100)
 /// the task's recursive dependencies. If this is set to true, then the system
 /// will attempt to reconstruct the object from its lineage if the object is
 /// lost.
-RAY_CONFIG(bool, lineage_pinning_enabled, true)
+RAY_CONFIG(bool, lineage_pinning_enabled, false)
```

Comment:
Could you add a comment to tell why we change this.

Reply:
Maybe we should link the origin github issue in the comment for further tracking.